NAS-102737 Rewrite of explanation of available/used space in pools. (#1523)
freenas/freenas-docs: userguide/storage.rst

:numref:`Figure %s <zfs_vol_fig>` shows the new *pool1*.

.. _pool capacity:

Select the pool to see more information. The first entry in the list
represents the root dataset and has the same name as the pool.

The :guilabel:`Available` column shows the estimated storage space
before
`compression <https://en.wikipedia.org/wiki/Data_compression>`__.
The :guilabel:`Used` column shows the estimated space used after
compression. These numbers come from :command:`zfs list`.

> Other utilities may show different numbers. For example, the available

Reviewer: ```suggestion
Other utilities can report different space estimates. For example, the available
```
Suggested making the sentence more specific.

> space shown in :command:`zpool status` is the cummulative space

Reviewer: ```suggestion
space shown in :command:`zpool status` is the cumulative space
```
spelling
